recipe/conda_build_config.yaml

```
@@ -1,2 +1,5 @@
 arm_variant_type:  # [aarch64]
   - sbsa           # [aarch64]
+c_stdlib_version:  # [linux]
+  - 2.17           # [linux and aarch64]
```

Looks like `linux_aarch64` needs to be updated to 2.28 based on [this CI failure]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1162853&view=logs&j=60238c3a-ba84-5ca5-39dc-717fbb421dbc&t=9cf66154-4ecc-5714-a3ee-b77d4233a250&l=944 )

```
Detected the system's glibc as 2.34
binary glibc 2.17.0 <= recipe glibc 2.17 lib/libcusolverMg.so.11.7.1.2
The binary is compatible.
binary glibc 2.27.0 <= recipe glibc 2.17 lib/libcusolver.so.11.7.1.2
The binary is not compatible with the recipe's glibc pinning.
```

```suggestion
  - 2.28           # [linux and aarch64]
```